src/snowflake/connector/constants.py

```diff
@@ -180,6 +180,9 @@ def struct_pa_type(metadata: ResultMetadataV2) -> DataType:
     FieldType(
         name="GEOMETRY", dbapi_type=[DBAPI_TYPE_STRING], pa_type=lambda _: pa.string()
     ),
+    FieldType(
+        name="FILE", dbapi_type=[DBAPI_TYPE_STRING], pa_type=lambda _: pa.string()
+    ),
```

Comment:
Better append it instead of insert it in the middle. It may change the index of each data type if added in the middle.

Reply:
ack